app/src/main/java/com/lagradost/cloudstream3/utils/BackupUtils.kt

```diff
@@ -283,4 +294,68 @@ object BackupUtils {
             setKeyRaw(it.key, it.value, isEditingAppSettings)
         }
     }
+
+
+    fun FragmentActivity.BackupGithub(){
+        val backup = this.getBackup()
+        ioSafe {
+            val tmpDir = createTempDir()
```

Comment:
might be good to clean up after yourself too, even if this is a temp dir

Reply:
tmpDir.delete() will do the cleaning right?

Reply:
check out https://stackoverflow.com/questions/4943629/how-to-delete-a-whole-folder-and-content

Reply:
I used tmpDir.deleteRecursively()(https://github.com/recloudstream/cloudstream/pull/47/commits/bdb6d78f97e966802e0330a16ab484204700dad7)